website/util/__init__.py

```diff
@@ -68,3 +73,23 @@ def is_json_request():
     """Return True if the current request is a JSON/AJAX request."""
     content_type = request.content_type
     return content_type and ('application/json' in content_type)
+
+
+def load_asset_paths():
+    if settings.DEV_MODE:
+        return {}
+    try:
+        return json.load(open('webpack-assets.json'))
+    except IOError:
+        logger.error('No "webpack-assets.json" file found. You may need to run webpack.')
+        raise
+
+
+asset_paths = load_asset_paths()
+base_static_path = '/static/public/js/'
+def webpack_asset(path):
```

Comment:
Could you please add a docstring to this function, indicating that it is meant to be used as a mako filter?